src/cli_plugin/install/index.js

```diff
@@ -18,6 +19,25 @@ function processCommand(command, options) {
   install(settings, logger);
 }
 
+function getDefaultConfigPath() {
+  const paths = [
+    fromRoot('config/kibana.yml'),
+    '/etc/kibana/kibana.yml'
+  ];
+
+  let defaultPath = paths[0];
+  paths.some(configPath => {
```

Comment:
Wouldn't using [_.find()](https://lodash.com/docs#find) make more sense than [_.some()](https://lodash.com/docs#some) here? They both iterate through a collection and stop once a condition is met, but... `some` is used to answer a question about the state of a collection; "Are there _some_ elements in the collection that meet these conditions?" Where `find` is used to return an element in a collection; "Give me the first element of a collection that meets these conditions"